src/rules/no-text-as-attribute.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'ensure that no plain text is used in attributes',
+      catefory: 'Possible errors',
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          attributes: {
+            type: 'array',
```

Comment:
Maybe we should specify that it's an array of string?
```json
{
  "type": "array",
  "minItems": 1,
  "items": { "type": "string" },
  "uniqueItems": true
}
```

Reply:
yes you're right 👍

Reply:
Done !